frontend/src/app/projects/project-detail/project-users/project-user-settings.component.html

```diff
@@ -26,8 +25,8 @@ <h2>Project members</h2>
       </div>
     </div>
   </div>
-  <mat-card>
-    <mat-form-field id="search" appearance="outline">
+  <mat-card class="w-[400px] max-w-[85vh] !pb-0.5 pr-0">
```

Comment:
```suggestion
  <mat-card class="w-[400px] max-w-[85vw] !pb-0.5 pr-0">
```
The max-width has to depend on the width, not the height. The current change break the mobile view.

Reply:
Good catch!